tests/integration-tests/configs/released.yaml

```diff
@@ -60,12 +60,12 @@ test-suites:
           instances: {{ common.INSTANCES_DEFAULT_X86 }}
           oss: ["rhel8"]  # os must be different from ubuntu2004 to test os validation logic when wrong os is provided
           schedulers: ["slurm"]
-    test_create.py::test_create_wrong_pcluster_version:
-      dimensions:
-        - regions: ["ca-central-1"]
-          instances: {{ common.INSTANCES_DEFAULT_X86 }}
-          oss: ["alinux2"]
-          schedulers: ["slurm"]
+#    test_create.py::test_create_wrong_pcluster_version:
```

Comment:
Why are you commenting out this test?

Reply:
the commit descriptions have the details https://github.com/aws/aws-parallelcluster/pull/6327/commits